packages/x-components/src/x-modules/search/components/index.ts

```diff
@@ -3,7 +3,7 @@ export { default as BannersList } from './banners-list.vue';
 export { default as PartialQueryButton } from './partial-query-button.vue';
 export { default as PartialResultsList } from './partial-results-list.vue';
 export { default as Promoted } from './promoted.vue';
-export { default as PromotedLists } from './promoteds-list.vue';
+export { default as PromotedList } from './promoteds-list.vue';
```

Comment:
```suggestion
export { default as PromotedsList } from './promoteds-list.vue';
```

Reply:
Well. Technically _promoteds_ doesn't exist. But we are using it in more places, so I'll leave up to you @tajespasarela